strongbox-client/pom.xml

```diff
@@ -39,6 +39,14 @@
         <developerConnection>scm:git:git://github.com/strongbox/strongbox.git</developerConnection>
     </scm>
 
+    <properties>
+        <jacoco.outputDir>${project.build.directory}/jacoco</jacoco.outputDir>
+        <jacoco.out.ut.file>jacoco-ut.exec</jacoco.out.ut.file>
```

Comment:
No need for a separate properties for these two:
* `jacoco.out.ut.file`
* `jacoco.out.it.file`